content/influxdb/clustered/reference/release-notes/influxctl.md

```diff
@@ -12,6 +12,30 @@ weight: 202
 canonical: /influxdb/cloud-dedicated/reference/release-notes/influxctl/
 ---
 
+## v2.6.0 {date="2024-03-18"}
+
+`influxctl` 2.6.0 introduces the ability to create, list, and revoke
+management tokens. These tokens allow users to authenticate directly without the
+need to for OAuth2.
```

Comment:
```suggestion
`influxctl` 2.6.0 introduces the ability to create, list, and revoke
management tokens and allow you to authenticate directly with your
InfluxDB cluster instead of an OAuth2 provider.
```